### ISSUE TITLE
Make a plan for importing various "Reports about the FEC"

### ISSUE THREAD

**Issue by noahmanger** [18F/fec-cms#779]
The "Reports about the FEC" section in "About" calls for several different pages, each of which acts as a feed of the various reports about the agency (IG reports, budget docs, etc.). Many of these are PDFs, with an occasional HTML page. 

I'd like to use this issue to:
- [x] Identify all of the reports and where they will live (likely in a spreadsheet)
- [x] Determine what we need to do on the back-end to support these documents
- [x] Determine how we will add them to the system

**Comment by noahmanger** [comment]
Progress update on a suggested approach over in this PR https://github.com/18F/fec-cms/pull/804

**Comment by noahmanger** [comment]
@patphongs and @emileighoutlaw I made a [new spreadsheet](https://docs.google.com/spreadsheets/d/1ZIismo1y8mih7O4XoCTuyq13uKwtCp_Dz_RzMstOFQ4/edit#gid=0) :lock: that we can use for organizing these reports.

This has columns for all of the fields we need for the import script, it just needs to be filled out. I set up the category column to validate based on the sub-categories available for each parent category.

Can I hand this off to one of you to fill take from here?

**Comment by emileighoutlaw** [comment]
Want to divvy this up between the two of us, @patphongs ?

**Comment by patphongs** [comment]
@emileighoutlaw Sure let's look at this together. Let me talk to you about it.

**Comment by emileighoutlaw** [comment]
⚠️  💟  Long post warning. Thanks for sticking with me on this 💟  ⚠️  

**tl;dr** I've been working with the [spreadsheet of report content](https://docs.google.com/spreadsheets/d/1ZIismo1y8mih7O4XoCTuyq13uKwtCp_Dz_RzMstOFQ4/edit#gid=0) :lock:. I think we _may_ need to adjust our design. 

Current design pattern:
> Date
> ### Title
> Link (PDF | size)
> [ category tag ] 

🎉 Yay! So awesome! But with real content, the fields are regularly repetitive, a lá:

> October 3, 2011
> ### Office of Inspector General FY 2012 work plan
> Office of Inspector General FY 2012 work plan (PDF | 250KB)
> [ work plan ] 



😭  Not awesome. 

My first fix for this repetition was to tailor the link field to avoid duplication:
> October 3, 2011
> ### Office of Inspector General FY 2012 work plan
> Open PDF (250KB)
> [ work plan ] 

🎉  Yay again! 
😭  But there are sticky wickets here, too:
- When I ran that idea by our accessibility guild, I learned that screen readers often browse pages in a way where users just hear the link titles. So they'd get "Open PDF (250 KB)" without additional context. Their recommendastion was that we add a `.sr-only` span in the link that includes the title of the document. 
- The nicely tailored "Open PDF" content only works if it's something that happens automatically and isn't a field humans have to fill in. If we leave it up to well-intentioned folks to enter the link information, they'll likely (by accident and with the best intentions) end up creating repetitive link text. Or inconsistent links. 

So! That leads me to my questions.

- ❔ Is a way to automatically create a `.sr-only` span, which includes the document title?
- ❔Is there a way to automatically in the future populate the all links with the text "Open PDF (size)"? 

If yes and yes, this seems like a good solution. We can stop here. 

If no ⬇️ :

We should probably tweak the design. These feeds are going to house the content folks are **least** willing and able to adjust by hand. If we set folks up with multiple fields to fill out, we're paving the way for future duplication like this (h/t to Corey M. for sharing this case study from her California work):

![screen_shot_2017-02-15_at_1 54 54_pm_480](https://cloud.githubusercontent.com/assets/10552702/22999122/772d37e4-f39f-11e6-939d-42c375561b53.png)

That first link to salary scales is a repetition nightmare.

Basically, I looked at a lot of ways that we could tailor this content to make it more friendly to the design. But I think anything that has to be done by hand is going to be a mistake. Instead, I'd recommend we move to a design with fewer fields. Like this one:

<img width="746" alt="screen shot 2017-02-15 at 5 06 09 pm" src="https://cloud.githubusercontent.com/assets/10552702/22999453/14313f94-f3a1-11e6-92b8-54f69de6e58e.png">

The benefit of this one is that we have only:
> Date
> ### Title that is also a link (PDF | size)
> [ category tag ]


That's a giant bucket full of thoughts I just dumped here. So I'll give you time to digest @noahmanger. Also tagging @jenniferthibault, because this has to do with design. We can discuss in person, if we don't want to keep talking in GitHub.

**Comment by noahmanger** [comment]
@emileighoutlaw yeah, those solution's aren't tricky at all. We'd just have one field for document title and could include it twice in the template. I think I'd prefer the simpler approach that just shows the title with some indicator that it's a PDF. Though just noting that we won't have the "size" value unless you add it by hand.

**Comment by emileighoutlaw** [comment]
Cool. If Jen feels good about the solution, I think the best solution might be the most minimal one.

Using this as my guiding light:
<img width="746" alt="screen shot 2017-02-15 at 5 06 09 pm" src="https://cloud.githubusercontent.com/assets/10552702/22999453/14313f94-f3a1-11e6-92b8-54f69de6e58e.png">

And then with real content, the feed would end up looking something like this: 
> November 15, 2012
> ### Performance and accountability report (reports consolidation act of 2000) FY 2012 (PDF)
> [ Performance and accountability report ]


Even with this approach, there's going to be some content repetition, but I still think it's our best option without sinking a lot of time and energy into hand editing document information fields (which I am happy to do but don't think is sustainable long-term).

**Comment by jenniferthibault** [comment]
My only observation is that I'd like to keep the document icon associated with PDF files — this is how folks can tell which doc links will be a PDF and which will be an HTML page at a glance. 

**If it's a PDF:**
![screen shot 2017-02-16 at 2 54 26 pm](https://cloud.githubusercontent.com/assets/11636908/23038412/f19e034c-f457-11e6-8611-5c7d81c06bb8.png)

**If it's HTML:**
![screen shot 2017-02-16 at 2 54 49 pm](https://cloud.githubusercontent.com/assets/11636908/23038425/f96d1662-f457-11e6-81f9-90986b5c6d04.png)

**Comment by emileighoutlaw** [comment]
Our spreadsheet should be totally filled with content now, except for the few documents that we couldn't find dates for!

**Comment by emileighoutlaw** [comment]
Forgot to ping @noahmanger in ^^

**Comment by noahmanger** [comment]
Plan made and a new issue made for importing these #828 . I'll make the small change to the layout in a PR.
